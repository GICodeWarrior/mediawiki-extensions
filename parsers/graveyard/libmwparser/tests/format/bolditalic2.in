Normal text. '''''Bold italic text.''''' Normal text.
