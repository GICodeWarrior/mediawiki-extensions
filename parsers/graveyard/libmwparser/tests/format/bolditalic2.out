BEGIN ARTICLE
    BEGIN PARAGRAPH
        WORD[Normal]
        SPACE[ ]
        WORD[text]
        SPECIAL[.]
        SPACE[ ]
        BEGIN ITALIC
            BEGIN BOLD
                WORD[Bold]
                SPACE[ ]
                WORD[italic]
                SPACE[ ]
                WORD[text]
                SPECIAL[.]
            END BOLD
        END ITALIC
        SPACE[ ]
        WORD[Normal]
        SPACE[ ]
        WORD[text]
        SPECIAL[.]
    END PARAGRAPH
END ARTICLE
